shared/wallets/send-form/asset-input/index.stories.tsx

```diff
@@ -79,21 +79,93 @@ const StatefulAssetInput: any = withStateHandlers(
       return {value}
     },
   }
-)(AssetInput) as any
+)
+
+const StatefulAssetInputBasic = asStatefulInput(AssetInputBasic)
+/*
```

Comment:
what's up here?

Reply:
I haven't made much efforts to do stories yet; so just created PICNIC-135 for those.